mmf/trainers/core/training_loop.py

```diff
@@ -12,7 +12,7 @@
 from mmf.common.sample import to_device
 from mmf.utils.general import clip_gradients
 from torch import Tensor
-
+from mmf.utils.metsumm import metsumm
```

Comment:
this should go away, it's a debugging artifact. should not live in mmf codebase.